scipy-1.0/references.bib

```diff
@@ -20,3 +20,244 @@ @article{peterson2009f2py
+  month = jan,
+  year = {1994},
+  pages = {129--156}
+}
+
+@article{nelder_simplex_1965,
+  title = {A Simplex Method for Function Minimization},
+  volume = {7},
+  issn = {0010-4620},
+  timestamp = {2018-03-26T19:50:50Z},
+  number = {4},
+  journal = {The computer journal},
+  author = {Nelder, John A and Mead, Roger},
+  year = {1965},
+  pages = {308--313}
+}
+
+@article{powell_efficient_1964,
+  title = {An Efficient Method for Finding the Minimum of a Function of Several Variables without Calculating Derivatives},
+  volume = {7},
+  issn = {0010-4620},
+  timestamp = {2018-03-26T19:55:46Z},
+  number = {2},
+  journal = {The computer journal},
+  author = {Powell, Michael JD},
+  year = {1964},
+  pages = {155--162}
+}
+
+@article{polak_note_1969,
+  title = {Note Sur La Convergence de M{\'e}thodes de Directions Conjugu{\'e}es},
+  volume = {3},
+  issn = {0373-8000},
+  timestamp = {2018-03-26T19:58:14Z},
+  number = {16},
+  journal = {Revue fran{\c c}aise d'informatique et de recherche op{\'e}rationnelle. S{\'e}rie rouge},
+  author = {Polak, Elijah and Ribiere, Gerard},
+  year = {1969},
+  pages = {35--43}
+}
+
+@incollection{powell_direct_1994,
+  title = {A Direct Search Optimization Method That Models the Objective and Constraint Functions by Linear Interpolation},
+  timestamp = {2018-03-26T20:34:42Z},
+  booktitle = {Advances in Optimization and Numerical Analysis},
+  publisher = {{Springer}},
+  author = {Powell, Michael JD},
+  year = {1994},
+  pages = {51--67}
+}
+
+@article{nash_newton-type_1984,
+  title = {Newton-Type Minimization via the {{Lanczos}} Method},
+  volume = {21},
+  timestamp = {2018-03-26T20:33:05Z},
+  number = {4},
+  journal = {SIAM Journal on Numerical Analysis},
+  author = {Nash, Stephen G.},
+  year = {1984},
+  pages = {770--788}
+}
+
+@article{zhu_algorithm_1997,
+  title = {Algorithm 778: {{L}}-{{BFGS}}-{{B}}: {{Fortran}} Subroutines for Large-Scale Bound-Constrained Optimization},
+  volume = {23},
+  issn = {0098-3500},
+  timestamp = {2018-03-26T20:00:43Z},
+  number = {4},
+  journal = {ACM Transactions on Mathematical Software (TOMS)},
+  author = {Zhu, Ciyou and Byrd, Richard H and Lu, Peihuang and Nocedal, Jorge},
+  year = {1997},
+  pages = {550--560}
+}
+
+@article{byrd_limited_1995,
+  title = {A Limited Memory Algorithm for Bound Constrained Optimization},
+  volume = {16},
+  issn = {1064-8275},
+  timestamp = {2018-03-26T20:23:51Z},
+  number = {5},
+  journal = {SIAM Journal on Scientific Computing},
+  author = {Byrd, Richard H and Lu, Peihuang and Nocedal, Jorge and Zhu, Ciyou},
+  year = {1995},
+  pages = {1190--1208}
+}
+
+@article{lenders_trlib:_2016,
+  title = {Trlib: {{A}} Vector-Free Implementation of the {{GLTR}} Method for Iterative Solution of the Trust Region Problem},
+  timestamp = {2018-03-26T20:21:16Z},
+  journal = {arXiv preprint arXiv:1611.04718},
+  author = {Lenders, Felix and Kirches, Christian and Potschka, Andreas},
+  year = {2016}
+}
```

Comment:
The paper is now published, it would be great if you could update the reference:

```
  title         = {trlib: {A} vector-free implementation of the {GLTR} method for iterative solution of the trust region problem},
  author        = {F. Lenders and C. Kirches and A. Potschka},
  journal       = {Optimization Methods and Software},
  publisher     = {Taylor & Francis},
  year          = {2018},
  doi           = {10.1080/10556788.2018.1449842},
  url           = {https://doi.org/10.1080/10556788.2018.1449842}
```

Reply:
Thank you. I will update the reference.